src/ai/susi/server/api/cms/FiveStarRateSkillService.java

```diff
@@ -70,17 +70,12 @@ public ServiceResponse serviceImpl(Query call, HttpServletResponse response, Aut
         String skill_name = call.get("skill", null);
         File skill = SusiSkill.getSkillFileInLanguage(language, skill_name, false);
         String skill_stars = call.get("stars", null);
-        String access_token = call.get("access_token", null);
```

Comment:
The relevant string containing the sample query `http://localhost:4000/cms/fiveStarRateSkill.json?model=general&group=Knowledge&skill=aboutsusi&stars=3&access_token=6O7cqoMbzlClxPwg1is31Tz5pjVwo3` needs to be updated in this file now that the access_token is removed.

Reply:
Sample endpoint remains the same, just the nullity check on access_token in get parameters has been removed.

Reply:
@sudheesh001 Do you agree with my comment?

Reply:
Agree with this.